pom.xml

```diff
@@ -112,6 +112,16 @@
             <version>1.7</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-multibranch</artifactId>
```

Comment:
This dependency should not be here I think. Maybe you were looking for `SCMSourceOwner`?